app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -28,6 +29,10 @@ def host_subscriptions_names(host)
         host.subscriptions.map(&:name)
       end
 
+      def host_redhat_subcriptions_consumed(host)
+        host.pools.redhat.pluck(:consumed).sum
```

Comment:
You can 
```suggestion
        host.pools.redhat.sum(:consumed)
```
instead and let the DB do the calculation instead of pulling all rh pools to memory and summing it.